vms/registration/forms.py

```diff
@@ -5,7 +5,7 @@
 
 class UserForm(forms.ModelForm):
     # password not visible when user types it out
-    password = forms.CharField(widget=forms.PasswordInput())
+    password = forms.CharField(widget=forms.PasswordInput(), min_length=8, max_length=25)
```

Comment:
could you also add a message for the user that they won't be able to have a password < 8 length?